test/spec/modules/audiencerunBidAdapter_spec.js

```diff
@@ -0,0 +1,202 @@
+import { expect } from 'chai';
+import { spec } from 'modules/audiencerunBidAdapter';
+import { newBidder } from 'src/adapters/bidderFactory';
+
+const ENDPOINT = 'https://d.audiencerun.com/prebid';
+
+const BID_SERVER_RESPONSE = {
+  body: {
+    bid: [
+      {
+        'bidId': '51ef8751f9aead',
+        'zoneId': '12345abcde',
+        'adId': '1234',
+        'crid': '5678',
+        'cpm': 8.021951999999999999,
+        'currency': 'USD',
+        'w': 728,
+        'h': 90,
+        'isNet': false,
+        'buying_type': 'rtb',
+        'sync': [
+          'http://linksync1',
+          'http://linksync2'
+        ],
+        'adm': '<!-- test creative -->'
+      }
+    ]
+  }
+};
+
+describe('AudienceRun bid adapter tests', function() {
+  const adapter = newBidder(spec);
+
+  describe('inherited functions', function() {
+    it('exists and is a function', function() {
+      expect(adapter.callBids).to.exist.and.to.be.a('function');
+    });
+  });
+
+  describe('isBidRequestValid', function() {
+    let bid = {
+      'bidder': 'audiencerun',
+      'params': {
+        'zoneId': '12345abcde'
+      },
+      'adUnitCode': 'adunit-code',
+      'sizes': [[300, 250], [300, 600]],
```

Comment:
`sizes` is no longer supported in Prebid 3. `mediaTypes.banner.sizes` should be used instead.